app/src/androidTest/java/org/mozilla/fenix/helpers/TestAssetHelper.kt

```diff
@@ -5,16 +5,17 @@
 package org.mozilla.fenix.helpers
 
 import android.net.Uri
-import java.util.concurrent.TimeUnit
 import okhttp3.mockwebserver.MockWebServer
 import org.mozilla.fenix.helpers.ext.toUri
+import java.util.concurrent.TimeUnit
 
 /**
  * Helper for hosting web pages locally for testing purposes.
  */
 object TestAssetHelper {
     @Suppress("MagicNumber")
     val waitingTime: Long = TimeUnit.SECONDS.toMillis(15)
+    val downloadPromptWaitingTime = TimeUnit.SECONDS.toMillis(25)
```

Comment:
I have a similar increased loading waiting time in my collections tests PR, called pageLoadingTime, we should both name it something more generic: waitingTimeLong?

Reply:
Sounds good, but let's see if it works in this case as well.

Reply:
All good! Works as a charm. 🎉
✅ Successfully passed 100x on Firebase.